source/jormungandr/jormungandr/interfaces/v1/Places.py

```diff
@@ -359,4 +360,8 @@ def get(self, region=None, lon=None, lat=None, uri=None):
         self._register_interpreted_parameters(args)
         response = i_manager.dispatch(args, "places_nearby",
                                       instance_name=self.region)
+        response = marshal(response, places_nearby)
+        if args["bss_stands"]:
+            bss_manager = BssProviderManager()
```

Comment:
I think you should initializeit only once no ? you can put this in __init__.py in jormungandr